package.json

```diff
@@ -24,7 +24,8 @@
   "scripts": {
     "build": "tsc && downlevel-dts . ts3.4 && cp tsconfig.json ./ts3.4/",
     "build:babel-test": "babel src/test/lib/decorators_test.ts --out-file test/lib/decorators-babel_test.js",
-    "gen-docs": "typedoc --readme docs/_api/api-readme.md --tsconfig tsconfig_apidoc.json --mode modules --theme docs/_api/theme --excludeNotExported --excludePrivate --ignoreCompilerErrors --exclude '{**/*test*,**/node_modules/**,**/test/**}' --out ./docs/api --gaID UA-39334307-23 src/**/*.ts",
+    "gen-docs-test": "typedoc --readme docs/_api/api-readme.md --mode library --theme docs/_api/theme --excludeNotExported --excludePrivate --ignoreCompilerErrors --exclude '{**/*test*,**/node_modules/!(lit-html)/**,**/test/**}' --out ./docs/api --gaID UA-39334307-23 src/lit-element.ts src/lib/updating-element.ts",
```

Comment:
what's the difference between these two?

Reply:
Ahh... One belongs here, and the other I should've deleted before sending the PR 😁